instrumentation/spring/spring-kafka-2.7/library/build.gradle.kts

```diff
@@ -0,0 +1,21 @@
+plugins {
+  id("otel.library-instrumentation")
+}
+
+dependencies {
+  compileOnly("com.google.auto.value:auto-value-annotations")
+  annotationProcessor("com.google.auto.value:auto-value")
+
+  implementation(project(":instrumentation:kafka:kafka-clients:kafka-clients-common:library"))
+
+  compileOnly("org.springframework.kafka:spring-kafka:2.7.0")
+
+  testImplementation(project(":instrumentation:spring:spring-kafka-2.7:testing"))
+  testImplementation(project(":instrumentation:kafka:kafka-clients:kafka-clients-2.6:library"))
+
+  // 2.7.0 has a bug that makes decorating a Kafka Producer impossible
+  testImplementation("org.springframework.kafka:spring-kafka:2.7.1")
```

Comment:
I support the `spring-kafka-2.7` naming even though it only supports 2.7.1+

Reply:
It actually kinda does support 2.7 -- `spring-kafka-2.7` only instruments the consumer part, and the problem is within the producer-specific spring code 😄